### ISSUE TITLE
Fairness of comparison

### ISSUE THREAD

**Issue by semink** [songyangco/STNN#3]
The dataset seems different from the one you compared (DCRNN, GWNET, etc.).

Your dataset is "interpolated," so there are no "zero" values. I believe it's OK if you do it ONLY in the training set, but **the problem here is the test set is also manipulated**. In this case, it's not a fair comparison.

When I train and test the model with the one that is not interpolated, the error is comparably more significant than those reported in your paper.

**Comment by semink** [comment]
+others split the dataset by 70% training, 10% validation, and 20% test sets, but this work does 70%,20%,10%. It may produce different scores, so it should be fixed for fair comparison.

**Comment by semink** [comment]
For PEMS-BAY with correct setting, your model performs
Horizon 1 (5 min) - MAE: 1.74, RMSE: 2.95
Horizon 2 (10 min) - MAE: 1.94, RMSE: 3.30
Horizon 3 (15 min) - MAE: 2.05, RMSE: 3.56
Horizon 4 (20 min) - MAE: 2.26, RMSE: 3.89
Horizon 5 (25 min) - MAE: 2.35, RMSE: 4.10
Horizon 6 (30 min) - MAE: 2.50, RMSE: 4.34
Horizon 7 (35 min) - MAE: 2.71, RMSE: 4.60
Horizon 8 (40 min) - MAE: 2.82, RMSE: 4.80
Horizon 9 (45 min) - MAE: 2.90, RMSE: 4.95
Horizon 10 (50 min) - MAE: 3.04, RMSE: 5.15
Horizon 11 (55 min) - MAE: 3.16, RMSE: 5.35
Horizon 12 (60 min) - MAE: 3.28, RMSE: 5.53
Aggregation - MAE: 2.56, RMSE: 4.38

and for METR-LA
Horizon 1 (5 min) - MAE: 2.77, RMSE: 4.44
Horizon 2 (10 min) - MAE: 3.28, RMSE: 5.44
Horizon 3 (15 min) - MAE: 3.58, RMSE: 6.16
Horizon 4 (20 min) - MAE: 3.99, RMSE: 6.81
Horizon 5 (25 min) - MAE: 4.32, RMSE: 7.36
Horizon 6 (30 min) - MAE: 4.60, RMSE: 7.86
Horizon 7 (35 min) - MAE: 4.98, RMSE: 8.32
Horizon 8 (40 min) - MAE: 5.17, RMSE: 8.72
Horizon 9 (45 min) - MAE: 5.41, RMSE: 9.08
Horizon 10 (50 min) - MAE: 5.65, RMSE: 9.44
Horizon 11 (55 min) - MAE: 5.89, RMSE: 9.76
Horizon 12 (60 min) - MAE: 6.12, RMSE: 10.07
Aggregation - MAE: 4.65, RMSE: 7.79

**Comment by semink** [comment]
![interpolation](https://user-images.githubusercontent.com/21964698/148701336-ceeb8cc8-c4bd-4cc3-896a-9aeb1c9978b8.png)
A example of the interpolation in the test set. 
Blue: raw data for other SOTA methods.
Orange: raw data what you used.
